queries/tact/textobjects.scm

```diff
@@ -0,0 +1,278 @@
+; See: https://github.com/nvim-treesitter/nvim-treesitter-textobjects#built-in-textobjects
+; function.inner & outer
+; ----------------------
+; statis
+(static_function
+  body: (_)) @function.outer
+
+(static_function
+  body: (function_body
+    "{"
+    (_) @_start @_end
+    (_)? @_end
```

Comment:
This has the same issue, the pattern only matches the start and then an optional node following it. You may need anchors to ensure that the final optional `@_end` node is actually the end node: something like `. "{" . (_) @_start (_)? @_end . "}"` (also the `@_end` capture on the `@_start` capture is not needed)